cache_registry/manager.py

```diff
@@ -20,3 +21,41 @@ def check_integrity():
          for (e, ) in duplicates}
 
     pprint.pprint(d)
+
+
+@utils_manager.command
+def check_passed():
+    undertakings = Undertaking.query.all()
+    for undertaking in undertakings:
+        highleveluses = []
+        for businessprofile in undertaking.businessprofiles:
+            highleveluses.append(businessprofile.highleveluses)
+        types = []
+        for type in undertaking.types:
+            types.append(type.type)
+        contact_persons = []
+        for contact_person in undertaking.contact_persons:
+            contact_persons.append({
+                "userName": contact_person.username,
+                "firstName": contact_person.first_name,
+                "lastName": contact_person.last_name,
+                "emailAddress": contact_person.email
+            })
+        data = {
+            'id': undertaking.id,
+            'status': undertaking.status,
+            'businessProfile': {
+                'highLevelUses': highleveluses
+            },
+            'types': types,
+            'contactPersons': contact_persons,
+            'address': {
+                'country': {
+                    'type': undertaking.address.country.type
+                }
+            },
+            'euLegalRepresentativeCompany': undertaking.represent,
+            'domain': undertaking.domain
+        }
+        undertaking.check_passed = eea_double_check_fgases(data)
```

Comment:
check if undertaking is fgas or ods and call double_check_fgases or double_check_ods accordingly

Reply:
Done.